front/app/containers/Admin/projects/project/inputForm/IdeaFormBuilder/index.tsx

```diff
@@ -13,14 +13,12 @@ import { ideationConfig } from '../utils';
 const FormBuilder = lazy(() => import('components/FormBuilder/edit'));
 
 const IdeaFormBuilder = () => {
-  const { projectId, phaseId } = useParams() as {
+  const { projectId } = useParams() as {
```

Comment:
Since at this stage we're always using the custom fields on the project level for ideation, we can remove the phaseId here.